docs/source/traits_api_reference/trait_types.rst

```diff
@@ -238,10 +238,11 @@ Traits
 .. autoclass:: ToolbarButton
    :show-inheritance:
 
-.. autoclass:: Either
+.. autoclass:: Union
```

Comment:
Deliberately swapping the order so that `Union` appears before `Either` in the docs.

There's also a drive-by fix here: `:show-inheritance:` was missing for `Union`, for no reason that I can discern.